File: Questão-3/Java/ContaBancaria.java
public class ContaBancaria {
    private double saldo;
    public String titular;

    //Métodos
    public void depositar(double valor) {
        this.saldo += valor;
    }
    public void sacar(double valor) {
        if (this.saldo < valor) {
            System.out.println("Valor sacado ultrapassa saldo de "+this.saldo+" da sua conta");
        }
        else{
            this.saldo -= valor;
        }
    }
    //Construtor
    public ContaBancaria(double saldo, String titular) {
        this.saldo = saldo;
        this.titular = titular;
    }

    //Get e sets
    public double getSaldo() {
        return saldo;
    }
    public void setSaldo(double saldo) {
        this.saldo = saldo;
    }

    public String getTitular() {
        return titular;
    }

    public void setTitular(String titular) {
        this.titular = titular;
    }
}

File: Questão-3/ContaBancaria.py
class ContaBancaria:
    def __init__(self, saldo, titular):
      self.__saldo = saldo
      self.titular = titular

    def depositar(self, valor):
      self.__saldo += valor

    def sacar(self, valor):
      if self.__saldo < valor:
        print(f"valor sacado ultrapassa o saldo de {self.__saldo} da sua conta\n")
      else:
        self.__saldo -= valor

    def get_saldo(self):
      return self.__saldo


conta = ContaBancaria(5000, "João")
conta.depositar(1000)
conta.sacar(7000)
conta.sacar(2000)
print(f"A conta de {conta.titular} tem saldo de {conta.get_saldo()}")
